libjolie/src/main/java/jolie/lang/parse/OLParser.java

```diff
@@ -3329,6 +3331,13 @@ private OLSyntaxNode parseFactor()
 
 		if( path != null ) {
 			switch( token.type() ) {
+			case AT:
+				nextToken();
```

Comment:
Mmmh, what happens if one writes `x.y@Hello( "x" )`, which should be forbidden? It looks like you'll parse it incorrectly as x@Hello("x").